core/controllers/suggestion.py

```diff
@@ -1252,7 +1252,7 @@ def _construct_exploration_suggestions(
         content_html: Optional[Union[str, List[str]]] = None
         try:
             content_html = exploration.get_content_html(
-                suggestion.change.state_name, suggestion.change.content_id)
+            suggestion.change_cmd.state_name, suggestion.change_cmd.content_id)
```

Comment:
Why are you changing the indentation here? The previous indentation was correct -- see our coding style guide which talks about how to indent after a '('.

Please take a look at the Files Changed tab of your PR and make sure there are no spurious edits that shouldn't be committed.

Reply:
There was a Lint check failure occuring at this line due to indentation, due to which I changed the indentation in hopes of fixing it. I am sorry I didn't check the coding style guide before doing so. I reverted the indentation back to the original 

Thank you

Reply:
Done